windows/x_walletconfig_w.cpp

```diff
@@ -502,8 +554,77 @@ void WalletConfig::updateLogsStateUI(bool enabled) {
     ui->logsEnableBtn->setChecked(enabled);
 }
 
+void wnd::WalletConfig::on_logout_3_clicked()
+{
+    currentLogoutTimeout = 3 * 60;
+    updateButtons();
+}
+
+void wnd::WalletConfig::on_logout_5_clicked()
+{
+    currentLogoutTimeout = 5 * 60;
+    updateButtons();
+}
+
+void wnd::WalletConfig::on_logout_10_clicked()
+{
+    currentLogoutTimeout = 10 * 60;
+    updateButtons();
+}
 
+void wnd::WalletConfig::on_logout_20_clicked()
+{
+    currentLogoutTimeout = 20 * 60;
+    updateButtons();
 }
 
+void wnd::WalletConfig::on_logout_30_clicked()
+{
+    currentLogoutTimeout = 30 * 60;
+    updateButtons();
+}
 
+void wnd::WalletConfig::on_logout_never_clicked()
+{
+    currentLogoutTimeout = -1;
+    updateButtons();
+}
 
+void wnd::WalletConfig::on_start_mqs_clicked()
+{
+    updateButtons();
+}
+
+void wnd::WalletConfig::on_start_keybase_clicked()
+{
+    updateButtons();
+}
+
+void WalletConfig::updateAutoStartStateUI(bool isAutoStartMQS, bool isAutoStartKeybase) {
+    ui->start_mqs->setChecked(isAutoStartMQS);
+    ui->start_keybase->setChecked(isAutoStartKeybase);
+}
+
+void WalletConfig::updateAutoLogoutStateUI(int64_t time) {
+    if(time < 0)
+        ui->logout_never->setChecked(true);
+    switch(time) {
+        case 3 * 60:
+            ui->logout_3->setChecked(true);
+            break;
+        case 5 * 60:
+            ui->logout_5->setChecked(true);
+            break;
+        case 10 * 60:
+            ui->logout_10->setChecked(true);
+            break;
+        case 20 * 60:
+            ui->logout_20->setChecked(true);
+            break;
+        case 30 * 60:
+            ui->logout_30->setChecked(true);
+            break;
+    }
```

Comment:
time can be any value. Current default is 900 seconds and user always can change it in the config. We should handle that by selecting some reasonable value.  For 900 seconds you probably can select 20 minutes.